src/containers/Home/Jobs/ListJob/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React,{useEffect,useState} from 'react'
+import Job from '../../../model/job';
+import { getJobs } from '../../../Redux/jobs/jobSlice';
+import {useSelector , useDispatch} from "react-redux"
+import { RootState } from '../../../Redux/store/store';
+export default function ListJob() {
+    const data = useSelector ((state:RootState) => state.jobs)
```

Comment:
you can define like: 
`const {data: jobs, loading} = useSelector((state:RootState) => state.jobs)`
so you don't need lines 12, 13

Reply:
i've changed